integration/up_test.go

```diff
@@ -25,6 +26,223 @@ func (s *RunSuite) TestUpNotExistService(c *C) {
 	c.Assert(cn, IsNil)
 }
 
+func (s *RunSuite) TestRecreateForceRecreate(c *C) {
+	p := s.ProjectFromText(c, "up", SimpleTemplate)
+
+	name := fmt.Sprintf("%s_%s_1", p, "hello")
+	cn := s.GetContainerByName(c, name)
+	c.Assert(cn, NotNil)
+
+	p = s.FromText(c, p, "up", "--force-recreate", SimpleTemplate)
+	cn2 := s.GetContainerByName(c, name)
+	c.Assert(cn.ID, Not(Equals), cn2.ID)
+}
+
+func mountSet(slice []dockerclient.Mount) map[string]bool {
+	result := map[string]bool{}
+	for _, v := range slice {
+		result[fmt.Sprint(v.Source, ":", v.Destination)] = true
+	}
+	return result
+}
+
+func filter(s map[string]bool, f func(x string) bool) map[string]bool {
```

Comment:
This method looks useful enough to get into some `utils` package or something (with a unit test :heart_eyes_cat:.